hacktober-web/README.md

```diff
@@ -1,6 +1,18 @@
 # hacktober-web
 
```

Comment:
Right after your top-level is where you might want to put a purpose, high-level commentary, etc. It can be super brief (like a sentence or two), but by running straight into the prereqs without context -- even though we're in a subfolder of the main repo -- can be disorienting. For example, I have no idea that this is targeting Vue without seeing the source files or inspecting the hyperlinks.